capstone_project/src/main/webapp/scripts/maps_script.js

```diff
@@ -100,3 +100,86 @@ function createArticleEntry(text, schoolName) {
   node.appendChild(a);
   document.getElementById(schoolName).appendChild(node);
 }
+
+/** Creates a map centered at the geographic center of the US and
+ * adds it to the page. */
+function createMap() { // eslint-disable-line no-unused-vars
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 39.828502, lng: -98.579512}, zoom: 4});
+
+  // Create a marker for each school. (TODO: fetch this info from server)
+  createMarker(map, 33.639982, -117.844682, 'UCI');
+  createMarker(map, 47.655277, -122.303606, 'UW');
+  createMarker(map, 47.758821, -122.190725, 'UWB');
+}
+
+/** Creates and returns the content string for a marker.
+ * @param {string} name
```

Comment:
Here too, in general if you decide to have the @param and @return stuff you should describe what those are further than just the variable name.

If you don't think its necessary you could remove it also.

Reply:
Fixed @dchin2 I tried removing it in a previous pr but the linter gave me an error